File: {PWB}/Simple Programming/005. Simple Interest/C++/solution.cpp
#include<iostream>

using namespace std;

int main() {
   float amount, rate, time, si;

   cout<<"Enter Principal Amount : ";
   cin>>amount;

   cout<<"Enter Rate of Interest : ";
   cin>>rate;

   cout<<"Enter Period of Time   : ";
   cin>>time;

   si = (amount * rate * time) / 100;
   cout<<"\nSimple Interest : "<< si<<endl;

   return 0;
}

File: {PWB}/Simple Programming/005. Simple Interest/Java/solution.java
import java.util.Scanner;

public class Main {
	public static void main(String args[]) {
        Scanner scanner = new Scanner(System.in);
        float amount,period,rate,si;

        System.out.println("Enter Principle Amount :");
        amount = scanner.nextFloat();
      
        System.out.println("Enter rate annually : ");
        rate = scanner.nextFloat();
        
        System.out.println("Enter time in years : ");
        period = scanner.nextFloat();

        si = (amount*period*rate)/100;

        System.out.println("Simple Interest is : " + si);
    }
}
